hotline/templates/users/user_detail.html

```diff
@@ -0,0 +1,32 @@
+{% extends 'base.html' %}
+{% block content %}
+<div class="pull-right">
+    <a class="btn btn-success" href="{% url 'users-home' %}">Home</a>
+    <a class="btn btn-success" href="{% url 'users-detail' user.pk %}">View Profile</a>
+    <a class="btn btn-success" href="{% url 'users-edit' user.pk %}">Edit Profile</a>
+    <a class="btn btn-success" href="{% url 'reports-list' %}">Search Reports</a>
+    <a class="btn" href="{% url 'logout' %}">Logout</a>
+</div>
+
+    <h3>Details</h3>
+    <p>Name: {% if object.prefix != "None" %}
+                {{ object.prefix }} 
+             {% endif %}
+             {{ object.first_name }} {{ object.last_name }} 
+             {% if object.suffix != "None" %}
+                {{ object.suffix }}
+             {% endif %}
+             </p>
+             
+    <p>Date joined: {{ object.date_joined }}</p>
+    <p>Photo: {{ object.photo }}</p>
+    <p>Biography: {{ object.biography }}</p>
+    <p>Affiliations: {{ object.affiliations }}</p>
+    
+    {% if current_user.is_staff or current_uiser.is_manager %}
```

Comment:
Typo here. I also got rid of the is_manager flag, since a manager is just someone who can login (i.e. is_active=True).